remoteTelegramCommands/service.go

```diff
@@ -53,3 +53,35 @@ func (s remoteCommand) Execute(update tgbotapi.Update) {
 	request := RemoteRequest{Message: update.Message.CommandArguments(), From: strconv.FormatInt(int64(update.Message.From.ID), 10)}
 	s.remote.Send(&request)
 }
+func (s *service) RegisterCommandLet(request *Request, response RemoteCommand_RegisterCommandLetServer) error {
+	s.telegram.RegisterCommandLet(searchChefRecipeReplyCommand(request.State, request.Nextstate, response))
+	for {
+		time.Sleep(10 * time.Minute)
+	}
+}
+type searchChefRecipeReply struct {
```

Comment:
This code cannot be here - A remote command should exist in the module to with the commandlet belongs.